cli/src/commands/config.rs

```diff
@@ -136,6 +136,11 @@ pub(crate) struct ConfigSetArgs {
 pub(crate) struct ConfigEditArgs {
     #[clap(flatten)]
     pub config_args: ConfigArgs,
+    /// Print the path to the config file instead of launching an editor
+    ///
+    /// A config file at that path may or may not exist.
+    #[arg(long, visible_alias = "path")]
+    print_path: bool,
```

Comment:
TODO: I decided it's probably better to have a dedicated `jj config path` command. It seems more intuitive-sounding and more discoverable.

Reply:
Done.